hub/core/chunk_engine.py

```diff
@@ -1315,6 +1315,23 @@ def read_sample_from_chunk(
             local_sample_index, cast=cast, copy=copy, decompress=decompress
         )
 
+    def _get_full_chunk(self, index):
```

Comment:
Would be nice to annotate the return type which is `bool` for this method

Reply:
sure